R/zzz.R

```diff
@@ -1,3 +1,8 @@
 .onLoad <- function(lib, pkg) {
   run_on_load()
 }
+
+# Define %||% for R < 4.4
+if (!exists("%||%", envir = baseenv())) {
+  `%||%` <- function(x, y) if (is.null(x)) y else x
+}
```

Comment:
Is this needed?
This package should import `%||%` from the rlang package.
https://github.com/posit-dev/plumber2/blob/59310a15dfad26a53203344a6a33432ef2944012/NAMESPACE#L165